kano/utils/hardware.py

```diff
@@ -90,6 +93,12 @@
         'cpu_profile': RPI_2_CPU_PROFILE,
         'performance': RPI_2_B_SCORE,
         'arch': "armv7"
+    },
+    RPI_3_KEY: {
+        'name': 'Raspberry Pi 3',
+        'cpu_profile': RPI_3_CPU_PROFILE,
+        'performance': RPI_3_SCORE,
+        'arch': "armv7"
```

Comment:
Actually it's v8. Not that this will make a difference, as I carefully made chromium conditional on !=v6